pkg/security/secl/generators/accessors/accessors.go

```diff
@@ -697,6 +697,14 @@ func (e *Event) GetFieldType(field eval.Field) (reflect.Kind, error) {
 		return reflect.Invalid, &eval.ErrFieldNotFound{Field: field}
 }
 
+func (e *Event) GetFieldTypes(field eval.Field) map[eval.Field]string {
+	return map[eval.Field]string{
+		{{range $Name, $Field := .Fields}}
```

Comment:
If you want to remove whitelines between iterations in template you can use `{{-` or/and `-}}` (see https://golang.org/pkg/text/template/#hdr-Text_and_spaces)